test/unit/resources/legacy_gelamtest.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * LegacyGelamTest: Supporting actor-style tests after it was cool.
```

Comment:
It was bittersweet, because the more I understood the innards of all that, the less urgent it felt to actually go through with all this. :)